cursorless-talon/src/actions/get_text.py

```diff
@@ -1,29 +1,32 @@
 from typing import Optional
 
-from talon import actions
+from talon import Module, actions
 
 from ..targets.target_types import CursorlessTarget
 
+mod = Module()
 
-def cursorless_get_text_action(
-    target: CursorlessTarget,
-    *,
-    show_decorations: Optional[bool] = None,
-    ensure_single_target: Optional[bool] = None,
-) -> list[str]:
-    """Get target texts"""
-    options: dict[str, bool] = {}
-
-    if show_decorations is not None:
-        options["showDecorations"] = show_decorations
-
-    if ensure_single_target is not None:
-        options["ensureSingleTarget"] = ensure_single_target
-
-    return actions.user.private_cursorless_command_get(
-        {
-            "name": "getText",
-            "options": options,
-            "target": target,
-        }
-    )
+
+@mod.action_class
+class Actions:
+    def cursorless_get_text(
+        target: CursorlessTarget,
+        show_decorations: Optional[bool] = None,
+        ensure_single_target: Optional[bool] = None,
+    ) -> list[str]:
```

Comment:
this api will probably be a bit surprising for the most common use case of getting text from a single target. And I believe it will be awkward to use from a `.talon` file. I wonder if we want to

1. Change this action to return a string, and not expose `ensure_single_target`; we just set it to `true` before passing along to extension
1. Add a separate action `cursorless_get_text_list` or `cursorless_get_text_multiple` which returns a list, and doesn't set `ensure_single_target`

wdyt?

Reply:
I like it. Updated!